include/deal.II/fe/fe_enriched.h

```diff
@@ -244,6 +244,18 @@ class FE_Enriched : public FiniteElement<dim, spacedim>
    * Otherwise one would have to use different instance of this class for each
    * disjoint enriched domain.
    *
+   * If you don't plan to use this feature, you can utilize C++11 lambdas to
+   * define dummy functions. Below is an example which uses two functions with
+   * the first element to be enriched and a single function with the second one.
+   * @code
+   * FE_Enriched<dim> fe
+   * (&fe_base,
+   * {&fe_1, &fe_2},
+   * {{[=] (const typename Triangulation<dim>::cell_iterator &) -> const Function<dim> *{return &fe_1_function1;},
+   *   [=] (const typename Triangulation<dim>::cell_iterator &) -> const Function<dim> *{return &fe_1_function2;}},
+   *  {[=] (const typename Triangulation<dim>::cell_iterator &) -> const Function<dim> *{return &fe_2_function;}}});
```

Comment:
I got ever so slightly confused for a second by the expression `*{return ...}` where I thought for a moment that the star applies to the thing in curly braces based on its proximity. That's of course not the case.

Can you add a space between star and brace to make this more obvious?